registry/storage/driver/gcs/gcs.go

```diff
@@ -68,6 +69,7 @@ type driverParameters struct {
 	client        *http.Client
 	rootDirectory string
 	chunkSize     int
+	sClient       *storage.Client
```

Comment:
I'd like to get this renamed to `gcsClient` if that's ok with you -- or even better: `gcs`.
The reasons are mostly I am mildly dyslexic so it's not unusual I misread `client` for `sClient`.

Ideally, if we want to be consistent across storage drivers, we should call it `GCS` -- see s3 driver https://github.com/distribution/distribution/blob/26163d82560f4dda94bd7b87d587f94644c5af79/registry/storage/driver/s3-aws/s3.go#L156